src/components/Input/HybridDropDown/HybridSelect.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import DropDownGroup from "../DropDown/DropDownGroup";
+import Select from "../Select/Select";
+
+const HybridSelectWrapper = styled.div`
```

Comment:
Can we move it, its own style file?

Reply:
This one line is the only line of css on the whole component. So i think its alright to be alongside rest of the component.

Reply:
Updated.